requirements/requirements_authentication.in

```diff
@@ -1,17 +1,17 @@
-social-auth-app-django
-tabulate
+social-auth-app-django<=5.4.1
```

Comment:
This is duplicated with, and matches, `requirements_all.txt`. I'm not sure if that's theoretically guaranteed (resolving dependencies in isolation, vs with other files), but if it's not I feel like that would suggest other contradictions which are a problem anyway.

Reply:
The app-specific .txt files get generated with passing `-c requirements_all.txt` to `pip`. Basically it generates the _all.txt file and then uses that for consistency in generated the app-specific .txt files.

Reply:
amazing.